proto/iamanager/v5/iamanager.proto

```diff
@@ -7,7 +7,7 @@ import "common/v1/common.proto";
 
 service IAMPublicService {
     rpc GetNodeInfo(google.protobuf.Empty) returns (NodeInfo) {}
-    rpc GetCert(GetCertRequest) returns (GetCertResponse) {}
+    rpc SubscribeCertChanged(SubscribeCertChangedRequest) returns (stream CertInfo) {}
```

Comment:
If it is supposed to receive initial certificate on subscribe then this method should be renamed to `SubscribeCert` as well as `SubscribeCertRequest`. But if keep `GetCert` method and having the following sequence:
- `GetCert`
- Connect
- `SubscribeCertChanged`
then I'm ok to keep `SubscribeCertChanged` name.

Reply:
Renamed to SubscribeCert